### DISCUSSION TITLE
Trademark use

### DISCUSSION THREAD

**Discussion by nothans** [neovim/neovim#24206] [General]
Hi. We are going to mention neovim in a GitHub README. Are there any rules on using the name? TM?

Thanks!

**Comment by otakutyrant** [comment]
Just consult its [LISCENSE](https://github.com/neovim/neovim/blob/master/LICENSE.txt)?

**Comment by justinmk** [comment]
As long as no third-party product is presented as being part of the Neovim project, it's fine to mention Neovim by name.
